### ISSUE TITLE
Verification Chest Detection

### ISSUE THREAD

**Issue by nameless4687** [Dylan-Zheng/Rise-of-Kingdoms-Bot#60]
Hi @Dylan-Zheng, thanks for your amazing bot. I have a request, can u add function that detect the verification box when it appear so bot can verify captcha sooner?

**Comment by migo2011** [comment]
did you able to have captcha solved by bot ? please tell in details what did you use , I have 2captcha ID but didn't work for me ,, always stuck with captcha and keeps without doing anything about it

**Comment by qbbyybyby** [comment]
> did you able to have captcha solved by bot ? please tell in details what did you use , I have 2captcha ID but didn't work for me ,, always stuck with captcha and keeps without doing anything about it

yes bot work smoothly, did u turn on 100% recognition in 2captcha or try set display to 720x1280?

**Comment by migo2011** [comment]
> > did you able to have captcha solved by bot ? please tell in details what did you use , I have 2captcha ID but didn't work for me ,, always stuck with captcha and keeps without doing anything about it
> 
> yes bot work smoothly, did u turn on 100% recognition in 2captcha or try set display to 720x1280?

I tried both , 720*1280 inside BlueStacks also tried 1280*720 I really don't know if that would change anything coz it should be the same .game already will take over the full wide screen .

about 2captcha I use 100% verification in that order of installation 
2 minimum attempts
2 maximum attempts 
1 match 

is it good or do I have to change things there ?? thanks for help 🙏

**Comment by TuanVN008** [comment]
> update :: it works thanks mate 🙏🙏🙏

Bot is ok bro
